adapters/httpapi_compact.c

```diff
@@ -1425,6 +1433,12 @@ HTTPAPI_RESULT HTTPAPI_SetOption(HTTP_HANDLE handle, const char* optionName, con
             }
         }
     }
+    else if (strcmp(OPTION_SET_TLS_RENEGOTIATION, optionName) == 0)
+    {
+        unsigned int tls_renegotiation = *(unsigned int*)value;
```

Comment:
This shall be `bool`. Please see comment in line 750.

Reply:
Fixed on the commit 98666e9b4d46e63d900d4abd1bc1591d3049b5a6